Bio/SeqIO/FastaIO.py

```diff
@@ -42,38 +43,48 @@ def SimpleFastaParser(handle):
     ('delta', 'CGCGC')
 
     """
-    # Skip any text before the first record (e.g. blank lines, comments)
-    # This matches the previous implementation where .readline() was used
-    for line in handle:
-        if line[0] == ">":
```

Comment:
From memory, ``line[0] == ">"`` was faster than ``line.startswith(">")`` but that is the kind of performance edge case which could have changed in Python 3.

Reply:
I was thinking of the situation where `line` has length 0 and `line[0]` doesn't exist. But anyway that cannot happen here. `line[0] == ">"` is still faster with python3.

Reply:
Yes, perhaps I should have put a comment about the empty string case not applying?

Thanks for checking the speed and reverting this.